pkg/auth/authenticator/password/ldappassword/ldap.go

```diff
@@ -88,20 +83,13 @@ func (a *Authenticator) getIdentity(username, password string) (authapi.UserIden
 		return nil, false, nil
 	}
 
-	// Make the connection
+	// Make the connection and bind to it if a bind DN and password were given
 	l, err := a.options.ClientConfig.Connect()
 	if err != nil {
 		return nil, false, err
 	}
 	defer l.Close()
 
-	// If specified, bind the username/password for search phase
```

Comment:
If someone is providing bind credentials to their client configuration, why would we expect to connect without binding?